packages/map-template/src/components/MapsIndoorsMap/MapsIndoorsMap.jsx

```diff
@@ -60,6 +62,7 @@ function MapsIndoorsMap(props) {
         const appUserRolesQueryParameter = queryStringParams.get('appUserRoles')?.split(',');
         const externalIDsQueryParameter = queryStringParams.get('externalIDs')?.split(',');
         const gmMapIdQueryParameter = queryStringParams.get('gmMapId');
+        const useMapProviderModule = queryStringParams.get('useMapProviderModule');
```

Comment:
Maybe use the convention above `useMapProviderModuleParameter` here. 
I can see that on this branch the `kioskOriginLocationId` is not following the convention either, but that has been fixed in another branch 😄